lib/probes/winston.js

```diff
@@ -70,13 +67,13 @@ module.exports = function (winston) {
     return winston;
   }
 
-  if (semver.gte(pkg.version, '3.0.0')) {
+  if (semver.gte(info.version, '3.0.0')) {
     if (typeof target.prototype.write === 'function') {
       shimmer.wrap(target.prototype, 'write', patchWrite);
     } else {
       logMissing('Logger.prototype.write()');
     }
-  } else if (semver.gte(pkg.version, '1.0.0') && semver.lt(pkg.version, '3.0.0')) {
+  } else if (semver.gte(info.version, '1.0.0') && semver.lt(info.version, '3.0.0')) {
```

Comment:
`&& semver.lt(info.version, '3.0.0')` seems redundant at this point as it won't get here if it is >= 3.0.0

Reply:
good catch.